src/web/src/components/DepartmentList.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="d-flex flex-column flex-grow-1">
+    <div
+      v-for="(major, index) in majors"
+      :index="index"
```

Comment:
curious what does `:index` do?

Reply:
Huh, I tried removing it to show you the error I was getting when it wasnt there earlier, but now the error is gone. Classic. It was something along the lines of having a unique index attribute for each div helped vue access them. I just removed it as I guess its not needed now